Localization/en-US/Diseases/covid-19.po

```diff
@@ -1,15 +1,107 @@
 msgid ""
 msgstr ""
 "FAH-Translate-Schema-Version: 1.0\n"
-"URL: https://foldingathome.org/diseases/infectious-diseases/covid-19/\n"
+"URL: https://foldingathome.org/covid-19/\n"
```

Comment:
I think the URL used here should actually be the full one it redirects to, because that is what is going to be stored in the post metadata for wordpress I believe? This short link is more of a redirect. The reason for using the URL in the post would be that it makes more sense from an automation perspective for identifying the post corresponding to this file.

Reply:
The issue is that it is a website called https://foldingathome.org/covid-19/. It is private so if you are not signed in to wordpress you will not get redirected. But https://foldingathome.org/diseases/infectious-diseases/covid-19/ is the real one.